src/callback_handler/crypto.rs

```diff
@@ -16,17 +15,21 @@ struct CertificatePool {
 /// verify_certificate verifies the validity of the certificate, and if it is
 /// trusted with the provided certificate chain.
 /// If the provided certificate chain is empty, it is treated as trusted.
-pub fn verify_certificate(req: CertificateVerificationRequest) -> Result<BoolWithReason> {
+pub fn verify_certificate(
```

Comment:
Why do we need to drop the usage of the `CertificateVerificationRequest`?
I would prefer to keep it, makes book-keeping easier.

Reply:
I would like to remove the dependency from the sdk. It looks like a inversion of dependency. Maybe have a type internally to the evaluator to group the data?

Reply:
Removing `CertificateVerificationRequest` import is not going to help with that, since the `Certificate` type (which is now one of the arguments of the function) is imported from our SDK.

I don't see a problem with having `policy-evaluator` import the client SDK. We did that on purpose to reuse the same types the consumers of the crate are using (and the crate itself sends down serialized over waPC).

BTW, this function is public only inside of this crate, hence we can change its signature without worrying about the consequences for kwctl and policy-server.